open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/kev/KevCatalog.java

```diff
@@ -25,6 +26,7 @@
 import java.time.ZonedDateTime;
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
```suggestion
```

Reply:
KEV does not disallow additional properties in schema so retained them: https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities_schema.json